src/ORM/Behavior/TreeBehavior.php

```diff
@@ -76,6 +78,15 @@ class TreeBehavior extends Behavior
     ];
 
     /**
+     * {@inheritDoc}
+     */
+    public function initialize(array $config)
+    {
+        $this->_config['leftField'] = new IdentifierExpression($this->_config['left']);
+        $this->_config['rightField'] = new IdentifierExpression($this->_config['right']);
```

Comment:
It is not really trivial to override method inside trait. May be more sense implement getter method that retruns wrapped left or right based of its argument on behavior level?